recipes/seismic-rna/meta.yaml

```diff
@@ -0,0 +1,47 @@
+---
+
+about:
+  home: https://github.com/rouskinlab/seismic-rna
+  license: GPL-3.0
+  license_family: GPL
+  license_file: LICENSE
+  license_url: https://www.gnu.org/licenses/gpl-3.0.html
+  summary: SEISMIC-RNA software by the Rouskin Lab
+build:
+  noarch: python
+  number: 0
+  run_exports:
+  - '{{ pin_subpackage("seismic-rna", max_pin="x.x") }}'
+channels:
+- bioconda
+- conda-forge
+package:
+  name: seismic-rna
+  version: 0.19.2
+requirements:
+  build:
+  - python >=3.10
+  - hatch >=1.12
+  run:
+  - python >=3.10
+  - bowtie2 >=2.5.1
+  - fastqc >=0.12.1
+  - rnastructure >=6.3
+  - samtools >=1.17
+  - brotli >=1.0
+  - click >=8.1
+  - cutadapt >=4.4
+  - fastqsplitter >=1.2
+  - matplotlib-base >=3.6
+  - numpy >=1.23,<1.27
+  - numba >=0.59
+  - pandas >=1.5
+  - plotly >=5.11
+  - pyyaml >=6.0
+  - scipy >=1.9
```

Comment:
```suggestion
  - scipy >=1.9
  - brotli
```

Reply:
Hi @martin-g, thank you for editing my first submission to Bioconda. I'm confused what this change would do. It looks like line 41 is unchanged, while `- brotli >= 1.0` already appears in line 31, so why should I add `- brotli` to line 42?

Reply:
Ah! I missed it at line 31!
I've added it because the tests fail with `ModuleNotFoundError: No module named 'brotli'`
It must be something else. Please revert my change while investigating the issue!

Reply:
I solved the problem. Turns out that `conda install brotli` installs only a binary, not an importable Python module. The latter requires `conda install brotli-python`. It's confusing because `pip install brotli` (which I'd been using previously) does install an importable Python module. So it's one of those cases where `pip` and `conda` use different names for the same package -- as well as, confusingly, the same name for different packages.